packages/blade/package.json

```diff
@@ -108,6 +108,7 @@
   "dependencies": {
     "@babel/runtime": "7.16.5",
     "@types/body-scroll-lock": "3.1.0",
+    "@floating-ui/react": "0.24.2",
```

Comment:
We will need to add this in peer dep also?

Reply:
Added

Reply:
Removed lol. Forgot this was not needed on native. We add it when its needed on native too.